test/integration/features/basic.feature

```diff
@@ -8,6 +8,19 @@ Feature: Basic
      Then Minishift should have state "Running"
       And Minishift should have a valid IP address
 
+     # Check if developer have sudo permission then oc should output as below
+     # Since we can't determine the vm ip before had so just make sure `Name`
+     # should be part of output string
+     # NAME           STATUS    AGE
+     # 192.168.64.2   Ready     1m
+     When executing "oc --as system:admin get node"
```

Comment:
This is really a indirect test. It makes several assumptions which are not clear, for example that a non sudoer user cannot view this. I think we should add a couple of things here. The direct test imo is `oc get cluster role sudoer` needs to return sudoer. 

If we want we can than also keep this test, but would also add the opposite. `When executing "oc  get node" where the output is "User "developer" cannot list all nodes in the cluster" and the exit status 1